packages/make-angular-cli-faster/src/utilities/package-manager.ts

```diff
@@ -0,0 +1,93 @@
+import {
+  getPackageManagerCommand,
+  readJsonFile,
+  writeJsonFile,
+} from '@nrwl/devkit';
+import { execSync } from 'child_process';
+import { copyFileSync, existsSync, unlinkSync, writeFileSync } from 'fs';
+import { appRootPath } from 'nx/src/utils/app-root';
+import { sortObjectByKeys } from 'nx/src/utils/object-sort';
+import { dirname, join } from 'path';
+import { gte, lt, major } from 'semver';
+import { dirSync } from 'tmp';
+import { MigrationDefinition } from './types';
+
+// version when the Nx CLI changed from @nrwl/tao & @nrwl/cli to nx
+const versionWithConsolidatedPackages = '13.9.0';
+
+export function installDependencies(
```

Comment:
The functions in this file seem like it could probably come from elsewhere in the repo? Likely the `'nx'` package.

Reply:
They don't exist in `nx` at the moment. Are you saying I should create them there instead?

Reply:
Only the `checkForNPMRC` exists in the `migrate.ts` file but it's not exported. The `installDependencies` is unique to this package, it shouldn't come from anywhere else and I didn't find any function like `resolvePackageVersion` anywhere. Its implementation is based on a different function that exists in `migrate.ts` but they are different.

I could move the `checkForNPMRC` and `resolvePackageVersion` functions to `nx/src/shared/package-manager.ts` and export them there.